src/repr/src/relation.rs

```diff
@@ -925,6 +925,11 @@ impl RelationDesc {
         &self.typ
     }
 
+    /// Returns the owned relation type underlying this relation description.
+    pub fn into_typ(self) -> SqlRelationType {
```

Comment:
```suggestion
    pub fn into_type(self) -> SqlRelationType {
```

No need to mutilate the term here :)